apps/react-connect/src/app/page.tsx

```diff
@@ -0,0 +1,70 @@
+'use client';
+import Image from 'next/image';
+
+import { useConnect } from '@portabl/react-connect-with-portabl';
+import { useEffect, useState } from 'react';
+
+export default function Web() {
+  const { isLoading, isAuthenticated, loginWithRedirect, getAccessTokenSilently, logout } = useConnect();
+  const [isMounted, setIsMounted] = useState(false);
+  const [accessToken, setAccessToken] = useState<string | null>(null);
+
+  useEffect(() => setIsMounted(true), []);
```

Comment:
Question: this is to handle next hydration?

Reply:
Originally yes, it was to handle a mismatch -- now the default loading state has been reworked so we can remove this.